apps/internal-ui/src/components/Chip/types/index.ts

```diff
@@ -0,0 +1,19 @@
+import { ButtonHTMLAttributes, HTMLAttributes } from 'react';
+
+import { ColorVariants } from '@/variants/color/types';
+
+export interface ChipProps
+  extends Pick<
+    ButtonHTMLAttributes<HTMLButtonElement>,
+    'onClick' | 'disabled' | 'className'
+  > {
+  label: string;
```

Comment:
```suggestion
  label: string;
```
q: 이전 디자인 시스템에서

- Label: label
- Typography: text
- button: content

로 각 label props 가 달랐던 부분에서 모두 content 로 통일하는 게 나을까요? 라는 의견 주셨던 걸로 기억하는데 
이 부분 현재 의견 어떠신지 궁금합니다~!

Reply:
Label 컴포넌트나 Button, Input 등은 content보다는 label이란 Prop 명칭이 어색하지 않고 오히려 맞다는 느낌인데 Typography는 label이 어색해서 제 생각에는 두가지 방향일 것 같습니다.

Typography: text
Label, Button, Input, etc.: label

만약 모두 통일하고 싶다면 모두 children을 받는 게 맞을 것 같습니다.
다만 이 방식을 이용하게 되면 접근성 지원 부분들에 대해서 어려운 부분이 있어 저는 위의 대안이 더 나을 것 같은데 어떻게 생각하시나요?

Reply:
Typography 는 현재 children 으로 받도록 되어 있어서 제외하고 나머지 label 로 통일하는 걸로 하면 될 것 같습니다!

Reply:
넵, 그럼 label로 통일하시죠.
컨벤션 업데이트 하겠습니다.

Reply:
ref. https://thebbodek.atlassian.net/wiki/spaces/SD/pages/352256035/React+JSX#Prop-Name